mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1427,6 +1474,16 @@ func resourceSqlDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 	return resourceSqlDatabaseInstanceRead(d, meta)
 }
 
+func maintenanceVersionDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
+	// Ignore the database version part and only compare the last part of the maintenance version which represents the release date of the version.
```

Comment:
Question: How do we want to handle the database version part? not suppress at all? like do we want to suppress the change such as "MYSQL_5_7_38 => MYSQL_5_7_37" (sorry, not sure if this is a valid example)

Reply:
I think we can ignore the database version part. The last part of the maintenance version (the last 14 characters) should be a strictly increasing value even when there is any change in db version part.